src/context.cc

```diff
@@ -7028,6 +7028,10 @@ bool Context::checkIfSkipPomodoro(TimeEntry *te) {
     return false;
 }
 
+bool Context::IsTimelineUiEnabled() {
+    return user_->timeline_ui_flag;
```

Comment:
`user_` is not checked for `nullptr` here, which can cause a crash. This is currently reproducible when starting the app while logged out.

This also uncovers the issue that `TimerEntryListViewViewModel`'s lifetime is bigger than the `user_`'s lifetime. We should probably initialize the viewmodel's properties or the entire viewmodel on some library event, e.g. `Toggl.OnLogin` which should fire only when user gets logged in.